apps/paf/behaviours/set-navigation-section.js

```diff
@@ -75,7 +75,7 @@ module.exports = superclass => class extends superclass {
         }
       }
 
-
+      
```

Comment:
can you remove the extra whitespace please?

Reply:
done